pyproject.toml

```diff
@@ -10,6 +10,7 @@ requires-python = '>=3.10'
 readme = 'README.md'
 
 dependencies = [
+    "click>=8.1.8",
```

Comment:
@matthewcornell can you give a quick justification for adding the additional dependency here?

Reply:
Sure. https://click.palletsprojects.com/en/stable/ is an established library for making Command Line Interfaces (CLIs).